src/genpy/rostime.py

```diff
@@ -226,6 +228,9 @@ def to_time(self):
         """
         return self.to_sec()
 
+    def __hash__(self):
+        return super(Time, self).__hash__()
```

Comment:
Ok, I see it now. Something weird seems to be happening, so in the absence of knowing the reason, reintroducing these lines makes sense!